pdf-bygger/src/main/kotlin/no/nav/pensjon/brev/pdfbygger/Application.kt

```diff
@@ -60,7 +79,10 @@ fun Application.module() {
         post("/compile") {
             val logger = call.application.environment.log
 
-            val result = call.receive<PdfCompilationInput>()
+            val request = withContext(Dispatchers.IO) {
```

Comment:
Jeg er fremdeles ikke helt sikker på om jeg er enig i at det er riktig å flytte lesing av body inn i Dispatchers.IO. Dette er uansett ekvivalent til å skrive `withContext(Dispatchers.IO) { call.receive<PdfCompilationInput>() }`, bortsett fra at man da også gjør deserialiseringen innen for Dispatchers.IO konteksten. Er du sikker på at det faktisk er mer effektivt å gjøre lesingen på den måten?

Reply:
Ikke så lett å få frem det jeg prøver å si. Eneste forskjellen på endringene du har gjort i kommunikasjonen mellom brevbaker og pdf-bygger er at du offloader lesing av body i Dispatchers.IO konteksten.

Reply:
revertert.